.eslintrc.json

```diff
@@ -16,8 +16,9 @@
         "createDefaultProgram": true
       },
       "extends": [
-        "plugin:@angular-eslint/ng-cli-compat",
-        "plugin:@angular-eslint/ng-cli-compat--formatting-add-on",
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended",
+        "plugin:@angular-eslint/recommended",
         "plugin:@angular-eslint/template/process-inline-templates"
```

Comment:
Configure eslint to use recommended rules for demo and libs.

Reply:
I usually generate new angular project, see how things are setup there, and copy the setup with as minimal adjustments needed to make future maintenance simpler.